### PR TITLE
compilation and cuda configuration fixes + vtk6 support

### PR DESCRIPTION
**Pull request by ahundt**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,8 +1,23 @@
-project(ObjRecRANSACRun)
-cmake_minimum_required(VERSION 2.8.3)
```
Comment (cpaxton) [inline CMakeLists.txt:2]: why?

File: src/BasicTools/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 project(BasicTools)
 cmake_minimum_required(VERSION 2.6)
+cmake_policy(SET CMP0053 OLD)
```
Comment (cpaxton) [inline src/BasicTools/CMakeLists.txt:3]: Not supported in default Ubuntu version of cmake, please remove

**Review by cpaxton — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (ahundt) [reply CMakeLists.txt:2]: warning about project name in ROS

File: src/BasicTools/CMakeLists.txt (hunk 2 above)
Comment (ahundt) [reply src/BasicTools/CMakeLists.txt:3]: shouldn't it just be a no-op in older versions?

File: CMakeLists.txt (hunk 1 above)
Comment (cpaxton) [reply CMakeLists.txt:2]: no, the cmake min version change

File: src/BasicTools/CMakeLists.txt (hunk 2 above)
Comment (cpaxton) [reply src/BasicTools/CMakeLists.txt:3]: Nope:
```
CMake Error at /home/chris.paxton/catkin_ws/src/ObjRecRANSAC/CMakeLists.txt:16 (cmake_policy):
  Policy "CMP0053" is not known to this version of CMake.


CMake Error at /home/chris.paxton/catkin_ws/src/ObjRecRANSAC/src/BasicToolsL1/CMakeLists.txt:3 (cmake_policy):
  Policy "CMP0053" is not known to this version of CMake.


CMake Error at /home/chris.paxton/catkin_ws/src/ObjRecRANSAC/src/BasicTools/CMakeLists.txt:3 (cmake_policy):
  Policy "CMP0053" is not known to this version of CMake.


CMake Error at /home/chris.paxton/catkin_ws/src/ObjRecRANSAC/src/ObjRecRANSAC/CMakeLists.txt:3 (cmake_policy):
  Policy "CMP0053" is not known to this version of CMake.


CUDA_TOOLKIT_ROOT_DIR not found or specified
CMake Error at /home/chris.paxton/catkin_ws/src/ObjRecRANSAC/s
```

Comment (cpaxton) [reply src/BasicTools/CMakeLists.txt:3]: also please resolve conflicts

File: CMakeLists.txt (hunk 1 above)
Comment (ahundt) [reply CMakeLists.txt:2]: Probably doesn't need to be a separate project. This was originally from a year or two ago so I don't remember for certain

**Pull request merged**